.betterer.results

```diff
@@ -1,3 +1,27 @@
 // BETTERER RESULTS V1.
-exports[`no hack comments`] = { timestamp: 1571097214906, value: `{"../packages/cli/src/cli.ts":[[152,7]],"../packages/cli/src/init.ts":[[2333,7]],"../packages/logger/src/logger.ts":[[228,7]]}` };
-exports[`no raw console.log`] = { timestamp: 1571097218313, value: `{"../packages/betterer/src/betterer.ts":[[5220,11],[5926,11],[5997,11]],"../packages/logger/src/logger.ts":[[481,11],[1279,11],[1306,11],[2174,11]]}` };
\ No newline at end of file
+exports[`no hack comments`] = {
+  timestamp: 1571444466341,
+  value: `{
+    "../packages/cli/src/cli.ts": [
+      [7, 2, 7, "RegExp match"]
+    ],
+    "../packages/cli/src/init.ts": [
+      [88, 4, 7, "RegExp match"]
+    ]
+  }`
+};
+exports[`no raw console.log`] = {
+  timestamp: 1571444467440,
+  value: `{
+    "../packages/betterer/src/betterer.ts": [
+      [198, 4, 11, "TSQuery match"],
+      [225, 2, 11, "TSQuery match"]
```

Comment:
Great job on this! It'll be _much_ easier to build tooling for.

Reply:
Speaking of tooling, I've been thinking about whether it would be possible to hook into VS Code extensions for these things, and get a different colour squiggle for betterer issues!?

Reply:
Yeah, for sure! I'm actually planning to build a GitHub bot w/ checks. Both would be good though, as it provides continuous feedback.

Reply:
Here's an example: https://github.com/hoovercj/vscode-extension-tutorial/blob/master/src/features/hlintProvider.ts